ui/component/viewers/videoViewer/index.js

```diff
@@ -34,12 +34,38 @@ const select = (state, props) => {
   const playingUri = selectPlayingUri(state);
   const collectionId = urlParams.get(COLLECTIONS_CONSTS.COLLECTION_ID) || (playingUri && playingUri.collectionId);
   const isMarkdownOrComment = playingUri && (playingUri.source === 'markdown' || playingUri.source === 'comment');
+  const isClaimPlayable = (uri) => {
```

Comment:
I think if we've gotten to videoViewer, we've should already know if it's playable.

Reply:
Some logic may also go in `page/show/view.jsx` which selects the first collection item and redirects there.